src/components/common/GlobalContext.ts

```diff
@@ -1,5 +1,6 @@
 import { createContext, Dispatch, SetStateAction, useContext } from "react";
 
+// TODO: Should we separate the loading state from the unknown state? I don't like that we're checking the `member` property for null. Checking member + admin could be bundled together, in my opinion.
```

Comment:
While we may want to leave open the possibility of rendering different things to non-members vs members, I don't see why we'd need `null` for this, as it can be achieved by letting the initial state be false, and would only be set to true after successfully logging in. Only annoying thing about this change would be some refactoring, such as 

https://github.com/acmutsa/Portal/blob/c76c4a596609949eefc2ed732b92dc85c44b35d2/src/components/Navbar.tsx#L109C15-L109C15

You mention wanting to separate the loading state from the unknown state, and to clarify, are you talking about how member and admin are null even if they've submitted their sign-on but still in a loading phase? If I'm understanding this right, I can't think of a reason why we'd need to have a loading state for either of these, but there could definitely be something I'm not thinking of.